README.md

```diff
@@ -1297,11 +1299,10 @@ altogether that is contractually linked to the value of stock.)
 - RSUs are often considered less preferable to grantees since they remove control over when
   you owe tax. Options, if granted with an exercise price equal to the fair market value of
   the stock, are not taxed until exercise, an event under the control of the optionee.
-  If a company awards you an RSU or restricted stock award which vests over time, you will
-  be taxed on the vesting schedule.
-  You have been put on “autopilot” with respect to the timing of the tax event.
-  This can be a really bad thing if on the date of vesting the shares are worth a lot and
-  consequently you owe a lot of tax.
+  If an employee is awarded an RSU or restricted stock award which vests over time, they
+  will be taxed on the vesting schedule;
+  they have been put on “autopilot” with respect to the timing of the tax event.
+  If the shares are worth a lot on the date of vesting, the tax burden can be significant.
 - ☝️ You don’t want to confuse *restricted stock units* with *restricted stock*, which
   typically refers to a [direct award of stock](how-equity-compensation-is-granted), a
```

Comment:
Invalid internal link. Add `#`

Actually it should go to `#restricted-stock-awards`.